diff -ru Output.original/HTML/Standard/AgentTicketQueue.dtl Output/HTML/Standard/AgentTicketQueue.dtl
--- Output.original/HTML/Standard/AgentTicketQueue.dtl	2006-11-23 20:58:24.000000000 +0100
+++ Output/HTML/Standard/AgentTicketQueue.dtl	2006-12-01 13:42:58.000000000 +0100
@@ -17,6 +17,7 @@
 </tr>
 <tr>
   <td class="menu">
+  <a href="/airt/incident.php">$Text{"AIRT incident overview"}</a>
   $Text{"Tickets shown"}: $Data{"Result"} -
   $Text{"Page"}: $Data{"PageNavBar"} -
   $Text{"Tickets available"}: $Data{"TicketsAvail"} -
diff -ru Output.original/HTML/Standard/AgentTicketQueueTicketView.dtl Output/HTML/Standard/AgentTicketQueueTicketView.dtl
--- Output.original/HTML/Standard/AgentTicketQueueTicketView.dtl	2006-11-23 20:58:24.000000000 +0100
+++ Output/HTML/Standard/AgentTicketQueueTicketView.dtl	2006-12-05 18:18:27.000000000 +0100
@@ -8,6 +8,8 @@
 # the enclosed file COPYING for license information (GPL). If you
 # did not receive this file, see http://www.gnu.org/licenses/gpl.txt.
 # --
+   <script type="text/javascript" src="/airt/otrs-airt.js"></script>
+
 <!-- start ticket -->
 <table border="0" width="100%" cellspacing="0" cellpadding="3">
 <tr>
@@ -99,31 +101,68 @@
   </td>
   <td width="25%" align="left" valign="top" class="PriorityID-$Data{"PriorityID"}">
    <p align="left">
+   <hr/>
+   <form  action="/airt/otrs.php" method="get">
+   <p><b>$Text{"AIRT"}:</b><br/>
+   <input type="hidden" name="tn" value="$Data{"TicketNumber"}">
+   IP address/host: <input type="text" name="ip" size="15"/><br/>
+   <input type="submit" name="action" value="Search"/>
+   <input type="submit" name="action" value="New Incident"/><br/>
+   <hr/>
+   </p>
+
+   <p>Incident IDs:<br/>
+   <div id="airt_output_$Data{"TicketNumber"}">$Text{"none"}</div><p/>
+   <hr/> 
+
+   <p>$Text{"Associate with AIRT incidents"}:<br/>
+   <table>
+   <tr align="right">
+      <td>&nbsp;</td>
+      <td><select id="incidentstatus_$Data{"TicketNumber"}" name="incidentno_s">
+   <option disabled selected value="-1">Select incident</option>
+   </select></td>
+   </tr>
+   <tr>
+      <td><i>or</i></td>
+      <td><input type="text" name="incidentno_t" width="15"/></td>
+   </tr>
+   </table>
+   <input type="submit" name="action" value="Link">
+   </p>
+   <hr/>
+
+   <p><b>$Text{"Quick-keys:"}</b><br/>
+   <input type="submit" name="action" value="Close"/>
+   </p>
+   <hr/>
+   </form>
+   <script language="JavaScript">loadXMLDoc($Data{"TicketNumber"});</script>
     <table border="0" cellspacing="0" cellpadding="1">
+#    <tr>
+#      <td><b>$Text{"State"}:</b></td>
+#      <td><font color="red"><div title="$Quote{"$Text{"$Data{"State"}"}"}">$Quote{"$Text{"$Data{"State"}"}","18"}</div></font></td>
+#    </tr>
+#    <tr>
+#      <td><b>$Text{"Priority"}:</b></td>
+#      <td><font color="red"><div title="$Quote{"$Text{"$Data{"Priority"}"}"}">$Quote{"$Text{"$Data{"Priority"}"}","18"}</div></font></td>
+#    </tr>
+#    <tr>
+#      <td><b>$Text{"Queue"}:</b></td>
+#      <td><font color="red"><div title="$QData{"Queue"}">$QData{"Queue","18"}</div></font></td>
+#    </tr>
     <tr>
-      <td><b>$Text{"State"}:</b></td>
-      <td><font color="red"><div title="$Quote{"$Text{"$Data{"State"}"}"}">$Quote{"$Text{"$Data{"State"}"}","18"}</div></font></td>
-    </tr>
-    <tr>
-      <td><b>$Text{"Priority"}:</b></td>
-      <td><font color="red"><div title="$Quote{"$Text{"$Data{"Priority"}"}"}">$Quote{"$Text{"$Data{"Priority"}"}","18"}</div></font></td>
-    </tr>
-    <tr>
-      <td><b>$Text{"Queue"}:</b></td>
-      <td><font color="red"><div title="$QData{"Queue"}">$QData{"Queue","18"}</div></font></td>
-    </tr>
-    <tr>
-      <td><b>$Text{"CustomerID"}:</b></td>
-      <td><a href="$Config{"CustomerDBLink"}" $Config{"CustomerDBLinkTarget"}>$Quote{"$Data{"CustomerID"}","18"}</a></td>
-    </tr>
-#    <tr  valign="top">
+#      <td><b>$Text{"CustomerID"}:</b></td>
+#      <td><a href="$Config{"CustomerDBLink"}" $Config{"CustomerDBLinkTarget"}>$Quote{"$Data{"CustomerID"}","18"}</a></td>
+#    </tr>
+##    <tr  valign="top">
 #      <td><b>$Text{"Owner"}:</b></td>
 #      <td><div title="$QData{"Owner"}">$QData{"Owner","18"}</div></td>
 #    </tr>
-    <tr>
-      <td><b>$Text{"Escalation in"}:</b></td>
-      <td>$Data{"TicketOverTime"}</td>
-    </tr>
+#    <tr>
+#      <td><b>$Text{"Escalation in"}:</b></td>
+#      <td>$Data{"TicketOverTime"}</td>
+#    </tr>
 # --
 # show TicketFreeKey1-2 and TicketFreeText1-2
 # --
@@ -172,30 +211,30 @@
 <!-- dtl:block:AgentIsCustomer -->
 
 <!-- dtl:block:AgentAnswer -->
-   <p align="left">
-     <b>$Text{"Compose Answer"} ($Text{"email"}):</b>
-     <br>
-     <table border="0">
-       <tr align="left">
-         <td>
-         $Data{"StdResponsesStrg"}
-         </td>
-       </tr>
-     </table>
-   </p>
-
-   <p align="left">
-     <b>$Text{"Contact customer"} ($Text{"phone"}):</b>
-     <br>
-     <table border="0">
-       <tr align="left">
-         <td>
-<dtl if ($Data{"AgentPhone"} ne "0") { $Data{"AgentPhoneStrg"} = "<li><a href="$Env{"Baselink"}Action=AgentTicketPhone&TicketID=$Data{"TicketID"}&QueueID=$Data{"QueueID"}" onmouseover="window.status='$Text{"Phone call"}'; return true;" onmouseout="window.status='';">$Text{"Phone call"}</a></li>"; }>
-$Data{"AgentPhoneStrg"}
-         </td>
-       </tr>
-     </table>
-   </p>
+#   <p align="left">
+#     <b>$Text{"Compose Answer"} ($Text{"email"}):</b>
+#     <br>
+#     <table border="0">
+#       <tr align="left">
+#         <td>
+#         $Data{"StdResponsesStrg"}
+#         </td>
+#       </tr>
+#     </table>
+#   </p>
+#
+#   <p align="left">
+#     <b>$Text{"Contact customer"} ($Text{"phone"}):</b>
+#     <br>
+#     <table border="0">
+#       <tr align="left">
+#         <td>
+#<dtl if ($Data{"AgentPhone"} ne "0") { $Data{"AgentPhoneStrg"} = "<li><a href="$Env{"Baselink"}Action=AgentTicketPhone&TicketID=$Data{"TicketID"}&QueueID=$Data{"QueueID"}" onmouseover="window.status='$Text{"Phone call"}'; return true;" onmouseout="window.status='';">$Text{"Phone call"}</a></li>"; }>
+#$Data{"AgentPhoneStrg"}
+#         </td>
+#       </tr>
+#     </table>
+#   </p>
 <!-- dtl:block:AgentAnswer -->
 # --
       </td>
@@ -215,7 +254,9 @@
        $Data{"MoveQueuesStrg"}
        <input class="button" type="submit" value="$Text{"Move"}">
        </form>
+
       </td>
+
     </tr>
     </table>
 
diff -ru Output.original/HTML/Standard/AgentTicketZoom.dtl Output/HTML/Standard/AgentTicketZoom.dtl
--- Output.original/HTML/Standard/AgentTicketZoom.dtl	2006-11-23 20:59:24.000000000 +0100
+++ Output/HTML/Standard/AgentTicketZoom.dtl	2006-12-05 17:57:15.000000000 +0100
@@ -8,6 +8,8 @@
 # the enclosed file COPYING for license information (GPL). If you
 # did not receive this file, see http://www.gnu.org/licenses/gpl.txt.
 # --
+   <script type="text/javascript" src="/airt/otrs-airt.js"></script>
+
 <!-- dtl:block:Header -->
 <!-- start ticket head $Data{"TicketNumber"} -->
 <table border="0" width="100%" cellspacing="0" cellpadding="3" cols="2">
@@ -25,6 +27,7 @@
 <tr>
   <td width="70%" class="menu">
 
+  <a href="/airt/incident.php">$Text{"AIRT incident overview"}</a> - 
 <!-- dtl:block:Menu -->
 <!-- dtl:block:MenuItemSplit -->
  -
@@ -131,9 +134,40 @@
 </div>
   </td>
   <td width="25%" align="left" valign="top" class="PriorityID-$Data{"PriorityID"}">
+
+   <form  action="/airt/otrs.php" method="get">
+   <p><b>$Text{"AIRT"}</b><br/>
+   <hr/>
+
+   <input type="hidden" name="tn" value="$Data{"TicketNumber"}">
+   IP address/host: <input type="text" name="ip" size="15"/>
+   <br/>
+   <input type="submit" name="action" value="Search"/>
+   <input type="submit" name="action" value="New Incident"/>
+   <p/>
+   <select id="incidentstatus_$Data{"TicketNumber"}" name="incidentno">
+   <option disabled selected>existing incidents</option>
+   </select><br/>
+   <input type="submit" name="action" value="Link"/> to incident <input type="text" name="incidentno" size="15"/><br/>
+   </p>
+
+   <hr/>
+   </p>
+   </p>
+
+   <p>Incident IDs:<br/>
+   <div id="airt_output_$Data{"TicketNumber"}">$Text{"none"}</div><p/>
+   </p>
+   <hr/>
+   <p><b>$Text{"Quick-keys"}:</b><br/>
+   <input type="submit" name="action" value="Close"/>
+  </form>
+   <script language="JavaScript" type="text/javascript">loadXMLDoc('$Data{"TicketNumber"}');</script>
+  <hr/>
+  <p/>
 # --
 # show ticket status (state, lock, ...)
-$Data{"TicketStatus"}
+#$Data{"TicketStatus"}
 # --
 
     <table border="0" cellspacing="0" cellpadding="1" width="95%">
@@ -142,66 +176,66 @@
 # --
 # show compose links
 <!-- dtl:block:AgentIsCustomer -->
-   <p align="left">
-     <b>$Text{"Your own Ticket"}:</b>
-     <br>
-     <table border="0">
-       <tr align="left">
-         <td>
-     <li><a href="$Env{"Baselink"}Action=AgentTicketCustomerFollowUp&TicketID=$Data{"TicketID"}" onmouseover="window.status='$Text{"Compose Follow up"}'; return true;" onmouseout="window.status='';">$Text{"Compose Follow up"}</a></li>
-         </td>
-       </tr>
-     </table>
-   </p>
+#   <p align="left">
+#     <b>$Text{"Your own Ticket"}:</b>
+#     <br>
+#     <table border="0">
+#       <tr align="left">
+#         <td>
+#     <li><a href="$Env{"Baselink"}Action=AgentTicketCustomerFollowUp&TicketID=$Data{"TicketID"}" onmouseover="window.status='$Text{"Compose Follow up"}'; return true;" onmouseout="window.status='';">$Text{"Compose Follow up"}</a></li>
+#         </td>
+#       </tr>
+#     </table>
+#   </p>
 <!-- dtl:block:AgentIsCustomer -->
 
 <!-- dtl:block:AgentAnswer -->
-   <p align="left">
-     <b>$Text{"Compose Answer"} ($Text{"email"}):</b>
-     <br>
-     <table border="0">
-       <tr align="left">
-         <td>
-         $Data{"StdResponsesStrg"}
-         </td>
-       </tr>
-     </table>
-   </p>
-
-   <p align="left">
-     <b>$Text{"Contact customer"} ($Text{"phone"}):</b>
-     <br>
-     <table border="0">
-       <tr align="left">
-         <td>
-<dtl if ($Data{"AgentTicketPhone"} ne "0") { $Data{"AgentPhoneStrg"} = "<li><a href="$Env{"Baselink"}Action=AgentTicketPhone&TicketID=$Data{"TicketID"}&QueueID=$Data{"QueueID"}" onmouseover="window.status='$Text{"Phone call"}'; return true;" onmouseout="window.status='';">$Text{"Phone call"}</a></li>"; }>
-$Data{"AgentPhoneStrg"}
-         </td>
-       </tr>
-     </table>
-   </p>
+#   <p align="left">
+#     <b>$Text{"Compose Answer"} ($Text{"email"}):</b>
+#     <br>
+#     <table border="0">
+#       <tr align="left">
+#         <td>
+#         $Data{"StdResponsesStrg"}
+#         </td>
+#       </tr>
+#     </table>
+#   </p>
+#
+#   <p align="left">
+#     <b>$Text{"Contact customer"} ($Text{"phone"}):</b>
+#     <br>
+#     <table border="0">
+#       <tr align="left">
+#         <td>
+#<dtl if ($Data{"AgentTicketPhone"} ne "0") { $Data{"AgentPhoneStrg"} = "<li><a href="$Env{"Baselink"}Action=AgentTicketPhone&TicketID=$Data{"TicketID"}&QueueID=$Data{"QueueID"}" onmouseover="window.status='$Text{"Phone call"}'; return true;" onmouseout="window.status='';">$Text{"Phone call"}</a></li>"; }>
+#$Data{"AgentPhoneStrg"}
+#         </td>
+#       </tr>
+#     </table>
+#   </p>
 <!-- dtl:block:AgentAnswer -->
 # --
 # show article links
 <!-- dtl:block:AgentArticleCom -->
-   <p>
-     <b>$Text{"Article"}</b>
-     <table border="0">
-       <tr align="left">
-         <td>
-     <a href="$Env{"Baselink"}Action=AgentTicketForward&TicketID=$Data{"TicketID"}&ArticleID=$Data{"ArticleID"}&QueueID=$Data{"QueueID"}" onmouseover="window.status='$Text{"Forward"}'; return true;" onmouseout="window.status='';">$Text{"Forward"}</a>
-# check if bounce should be shown
-    <dtl if ($Data{"ArticleType"} eq "email-external") { $Data{"ArticleBounceString"} = "Bounce"; }>
-    <dtl if ($Data{"SenderType"} ne "customer") { $Data{"ArticleBounceString"} = ""; }>
-    <a href="$Env{"Baselink"}Action=AgentTicketBounce&TicketID=$Data{"TicketID"}&ArticleID=$Data{"ArticleID"}&QueueID=$Data{"QueueID"}" onmouseover="window.status='$Text{"Bounce"}'; return true;" onmouseout="window.status='';">$Text{"$Data{"ArticleBounceString"}"}</a>
-# check if split should be shown
-    <dtl if ($Data{"SenderType"} eq "customer") { $Data{"ArticleSplitString"} = "Split"; }>
-
-    <a href="$Env{"Baselink"}Action=AgentTicketPhone&ArticleID=$Data{"ArticleID"}&QueueID=$Data{"QueueID"}" onmouseover="window.status='$Text{"Split"}'; return true;" onmouseout="window.status='';">$Text{"$Data{"ArticleSplitString"}"}</a>
-         </td>
-       </tr>
-     </table>
-   </p>
+#   <p>
+#     <b>$Text{"Article"}</b>
+#     <table border="0">
+#       <tr align="left">
+#         <td>
+#     <a href="$Env{"Baselink"}Action=AgentTicketForward&TicketID=$Data{"TicketID"}&ArticleID=$Data{"ArticleID"}&QueueID=$Data{"QueueID"}" onmouseover="window.status='$Text{"Forward"}'; return true;" onmouseout="window.status='';">$Text{"Forward"}</a>
+## check if bounce should be shown
+#    <dtl if ($Data{"ArticleType"} eq "email-external") { $Data{"ArticleBounceString"} = "Bounce"; }>
+#    <dtl if ($Data{"SenderType"} ne "customer") { $Data{"ArticleBounceString"} = ""; }>
+#    <a href="$Env{"Baselink"}Action=AgentTicketBounce&TicketID=$Data{"TicketID"}&ArticleID=$Data{"ArticleID"}&QueueID=$Data{"QueueID"}" onmouseover="window.status='$Text{"Bounce"}'; return true;" onmouseout="window.status='';">$Text{"$Data{"ArticleBounceString"}"}</a>
+## check if split should be shown
+#    <dtl if ($Data{"SenderType"} eq "customer") { $Data{"ArticleSplitString"} = "Split"; }>
+#
+#    <a href="$Env{"Baselink"}Action=AgentTicketPhone&ArticleID=$Data{"ArticleID"}&QueueID=$Data{"QueueID"}" onmouseover="window.status='$Text{"Split"}'; return true;" onmouseout="window.status='';">$Text{"$Data{"ArticleSplitString"}"}</a>
+#         </td>
+#       </tr>
+#     </table>
+#   </p>
 <!-- dtl:block:AgentArticleCom -->
 
 # --
@@ -223,6 +257,8 @@
        <input class="button" type="submit" value="$Text{"Move"}">
        </form>
    </p>
+   
+
   </td>
 </tr>
 <!-- dtl:block:Body -->
