diff -ur Output.original/HTML/Standard/AgentTicketQueue.dtl Output/HTML/Standard/AgentTicketQueue.dtl
--- Output.original/HTML/Standard/AgentTicketQueue.dtl	2006-11-23 20:58:24.000000000 +0100
+++ Output/HTML/Standard/AgentTicketQueue.dtl	2006-12-01 13:42:58.000000000 +0100
@@ -17,6 +17,7 @@
 </tr>
 <tr>
   <td class="menu">
+  <a href="/airt/incident.php">$Text{"AIRT incident overview"}</a>
   $Text{"Tickets shown"}: $Data{"Result"} -
   $Text{"Page"}: $Data{"PageNavBar"} -
   $Text{"Tickets available"}: $Data{"TicketsAvail"} -
diff -ur Output.original/HTML/Standard/AgentTicketQueueTicketView.dtl Output/HTML/Standard/AgentTicketQueueTicketView.dtl
--- Output.original/HTML/Standard/AgentTicketQueueTicketView.dtl	2006-11-23 20:58:24.000000000 +0100
+++ Output/HTML/Standard/AgentTicketQueueTicketView.dtl	2006-11-24 10:23:33.000000000 +0100
@@ -8,6 +8,8 @@
 # the enclosed file COPYING for license information (GPL). If you
 # did not receive this file, see http://www.gnu.org/licenses/gpl.txt.
 # --
+   <script type="text/javascript" src="/airt/otrs-airt.js"></script>
+
 <!-- start ticket -->
 <table border="0" width="100%" cellspacing="0" cellpadding="3">
 <tr>
@@ -215,7 +217,23 @@
        $Data{"MoveQueuesStrg"}
        <input class="button" type="submit" value="$Text{"Move"}">
        </form>
+
+   <hr/>
+   <form  action="/airt/otrs.php" method="get">
+   <p><b>$Text{"AIRT"}:</b><br/>
+   <script language="JavaScript">loadXMLDoc($Data{"TicketNumber"});</script>
+   <input type="hidden" name="tn" value="$Data{"TicketNumber"}">
+   IP address/host: <input type="text" name="ip" size="15"/><br/>
+   <input type="submit" name="action" value="Search"/>
+   <input type="submit" name="action" value="New Incident"/><br/>
+   </p>
+
+   <p>Incident IDs:<br/>
+   <div id="airt_output">$Text{"none"}</div><p/>
+   </p>
+   </form>
       </td>
+
     </tr>
     </table>
 
diff -ur Output.original/HTML/Standard/AgentTicketZoom.dtl Output/HTML/Standard/AgentTicketZoom.dtl
--- Output.original/HTML/Standard/AgentTicketZoom.dtl	2006-11-23 20:59:24.000000000 +0100
+++ Output/HTML/Standard/AgentTicketZoom.dtl	2006-12-01 14:14:09.000000000 +0100
@@ -8,6 +8,8 @@
 # the enclosed file COPYING for license information (GPL). If you
 # did not receive this file, see http://www.gnu.org/licenses/gpl.txt.
 # --
+   <script type="text/javascript" src="/airt/otrs-airt.js"></script>
+
 <!-- dtl:block:Header -->
 <!-- start ticket head $Data{"TicketNumber"} -->
 <table border="0" width="100%" cellspacing="0" cellpadding="3" cols="2">
@@ -25,6 +27,7 @@
 <tr>
   <td width="70%" class="menu">
 
+  <a href="/airt/incident.php">$Text{"AIRT incident overview"}</a> - 
 <!-- dtl:block:Menu -->
 <!-- dtl:block:MenuItemSplit -->
  -
@@ -223,6 +226,33 @@
        <input class="button" type="submit" value="$Text{"Move"}">
        </form>
    </p>
+   
+<script language="JavaScript" type="text/javascript">loadXMLDoc('$Data{"TicketNumber"}');</script>
+
+  <hr/>
+   <form  action="/airt/otrs.php" method="get">
+   <p><b>$Text{"AIRT"}:</b><br/>
+
+   <input type="hidden" name="tn" value="$Data{"TicketNumber"}">
+   IP address/host: <input type="text" name="ip" size="15"/>
+   <br/>
+   <input type="submit" name="action" value="Search"/>
+   <input type="submit" name="action" value="New Incident"/>
+   <p/>
+   <input type="submit" name="action" value="Link"/> to incident <input type="text" name="incidentno" size="15"/><br/>
+   <!--
+   <select id="incidentstatus" name='incident'>
+    <option disabled selected>existing incidents</option>
+   </select>
+   -->
+   <br/>
+   </p>
+
+   <p>Incident IDs:<br/>
+   <div id="airt_output">$Text{"none"}</div><p/>
+   </p>
+  </form>
+
   </td>
 </tr>
 <!-- dtl:block:Body -->
